examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -6558,15 +6558,15 @@ endpoint 1 {
     ram      attribute currentScene default = 0x00;
     ram      attribute currentGroup default = 0x0000;
     ram      attribute sceneValid default = 0x00;
-    ram      attribute nameSupport;
+    ram      attribute nameSupport default = 0x80;
     ram      attribute lastConfiguredBy;
-    ram      attribute sceneTableSize;
-    callback attribute remainingCapacity;
+    ram      attribute sceneTableSize default = 16;
+    callback attribute remainingCapacity default = 8;
     callback attribute generatedCommandList;
     callback attribute acceptedCommandList;
     callback attribute eventList;
     callback attribute attributeList;
-    ram      attribute featureMap default = 0;
+    ram      attribute featureMap default = 3;
```

Comment:
Likewise, why is this stored in RAM at all if we always force values in the cluster?  Should we just be using the AttributeAccessInterface for this?